banking-bench/src/main.rs

```diff
@@ -134,7 +134,7 @@ fn make_accounts_txs(
                 hash,
                 compute_unit_price,
             );
-            let sig: Vec<u8> = (0..64).map(|_| thread_rng().gen::<u8>()).collect();
+            let sig: [u8; 64] = std::array::from_fn(|_| thread_rng().gen::<u8>());
```

Comment:
this pattern is pretty common. maybe we could add it as a helper behind @ryoqun's fancy new [`dev-context-only-utils` cargo feature](https://github.com/solana-labs/solana/pull/32169)?

Reply:
separate pr?